test/multiverse/suites/redis/redis_instrumentation_test.rb

```diff
@@ -336,8 +336,8 @@ def simulated_error_class
     end
 
     def simulate_read_error
-      redis = Redis.new(:host => redis_host)
-      redis.send(client).stubs("establish_connection").raises(simulated_error_class, "Error connecting to Redis")
```

Comment:
The stubbed method was changed to `connect` because `establish_connection` does not exist in Redis 5.0